### PR TITLE
SASS-6287- content for payments into pensions page

### PR DESCRIPTION
**Pull request by rhysconnor**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

##### Before creating PR
- [x]  Have you run the tests?
- [ ]  Have you run the it tests?
- [ ]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of main?
- [ ]  Have you checked code coverage isn’t lower than previously?
- [ ]  Have you done the group review in the team?

##### After PRs been raised
- [ ]  Have you checked the PR Builder passes?

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?
- [x]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [ ]  Have you run the tests?
- [ ]  Have you run the it tests?
- [ ]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- [ ]  Have you checked for individual and agent content matching with the prototype? (where applicable)
- [ ]  Have you checked for welsh transaltion for individual and agent? (where applicable)
- [ ]  Have you checked for errors messages for individual and agent on both english and welsh? (where applicable)
- [ ]  Run wave (chrome extension) for accessibility (where applicable)

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?
- [ ]  Demo to Business Analyst

### CODE REVIEW

File: app/views/pensions/PaymentsIntoPensionsAgentView.scala.html [hunk 1]
```diff
@@ -14,18 +14,37 @@
...
+    link: Link,
+    paragraph: ParagraphBody
 )
 
 @(form: Form[_], mode: Mode, taxYear: Int)(implicit request: Request[_], messages: Messages)
 
+@overseasTransferCharge = @{ Call("GET", config.overseasTransferChargeUrl)}
+
+@insetContent1 = {
+    <p class="govuk-body">
```
Comment (rhysconnor) [inline app/views/pensions/PaymentsIntoPensionsAgentView.scala.html:42]: Unsure if this is needed, happy to remove lmk

File: app/models/pensions/PaymentsIntoPensions.scala [hunk 2]
```diff
@@ -60,7 +79,19 @@ object PaymentsIntoPensions extends Enumerable.Implicits {
     values.zipWithIndex.map {
       case (value, index) =>
         value match {
-          case ExclusiveOption => CheckboxItemViewModel(
+          case Uk => CheckboxItemViewModel(
+            content = Text(messages(s"paymentsIntoPensions.${value.toString}")),
+            fieldId = "value",
+            index = index,
+            value = value.toString
+          ).withHint(Hint(content = Text(messages(s"paymentsIntoPensions.agent.uk.hint"))))
```
Comment (rhysconnor) [inline app/models/pensions/PaymentsIntoPensions.scala:87]: start hint with uppercase and end in fullstop

File: app/models/pensions/PaymentsIntoPensions.scala [hunk 3]
```diff
@@ -26,21 +27,39 @@ sealed trait PaymentsIntoPensions
 
 object PaymentsIntoPensions extends Enumerable.Implicits {
 
-  case object Option1 extends WithName("option1") with PaymentsIntoPensions
-  case object Option2 extends WithName("option2") with PaymentsIntoPensions
-  case object ExclusiveOption extends WithName("exclusive") with PaymentsIntoPensions
+  case object Uk extends WithName("uk") with PaymentsIntoPensions
```
Comment (rhysconnor) [inline app/models/pensions/PaymentsIntoPensions.scala:30]: Update name for pensions

**Review by kannanramakrishnan — DISMISSED** [review]

**Review by ghousedev — DISMISSED** [review]

**Review by ghousedev — APPROVED** [review]

File: app/models/pensions/PaymentsIntoPensions.scala [hunk 4]
```diff
@@ -60,7 +79,19 @@ object PaymentsIntoPensions extends Enumerable.Implicits {
...
+            value = value.toString
+          ).withHint(Hint(content = Text(messages(s"paymentsIntoPensions.agent.uk.hint"))))
+          case Divider => CheckboxItemViewModel(
+            fieldId = "value",
+            index = index,
+            value = value.toString,
+            divider = messages(s"site.or")
+          )
+          case No => CheckboxItemViewModel(
             content = Text(messages(s"paymentsIntoPensions.agent.${value.toString}")),
```
Comment (basotia) [inline app/models/pensions/PaymentsIntoPensions.scala:95]: do we need agent here?

**Review by basotia — APPROVED** [review]

**Pull request merged**